js/adjust.js

```diff
@@ -11,10 +11,8 @@ function DynamicAlign() {
 
             // compute left side & match difference
             var $left_item = $("#" + $section_id_name + " .left-docs" + " h3#" + $text);
-            console.log($left_item);
```

Comment:
Can you please explain the purpose of the two changes in this file and how they fit in the scope of the PR?

Reply:
This line forces browser's console log to be spammed while resizing window. As a contribution is about encouraging developers to play around with browsers console and `steem` API I thought it's worth to not distract them with such unnecessary output. Hope to hear from you.